core/src/env/engine/off_chain/test_api.rs

```diff
@@ -299,6 +299,25 @@ where
     })
 }
 
+/// Initializes the whole off-chain environment.
+/// Uses a fresh off-chain environment instance!
+///
+/// # Note
+///
+/// - Initializes the off-chain environment with default values that fit most
+/// uses cases.
+/// - The off-chain environment _must_ be initialized before use.
+pub fn recreate_and_initialize_as_default<T>() -> Result<()>
```

Comment:
So this whole thing is basically resetting the off-chain environment instance to default values? Could we call this procedure `reset_to_defaults` then? :) Also we could potentially be more efficient by introducing some more robust API that either initializes or resets with default values.